### PR TITLE
[ISSUE #348] Setup automated workflows for greetings

### PR DESCRIPTION
**Pull request by SteveYurongSu**
ISSUE #348

By setuping the GitHub Greetings workflow, we can:

1. Accurately identify new users and contributors
2. Encourage new users and contributors
3. Attract new users and contributors to the WeChat Group and our mailing lists

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Welcome to the Apache EventMesh (incubating) community!!
+            We are glad that you are contributing by opening this issue. :D
+
+            Please make sure to include all the relevant context.
+            One of the @apache/eventmesh-committers will be here shortly.
```
Comment (kezhenxu94) [inline .github/workflows/greetings.yml:36]: I don’t think it’s a good idea to mention all eventmesh committers in every new issue from first contributors. It’s kind of overwhelming

Comment (SteveYurongSu) [reply .github/workflows/greetings.yml]: Thanks for reviewing. I think you are right. I've removed the `@` in the content. :)

**Comment by SteveYurongSu** [comment]
Hi @xwm1992 @qqeasonchen, please review this PR and let me know if you have any questions. Thanks.

**Comment by xwm1992** [comment]
Hi @SteveYurongSu I want to know that is this greeting action will be triggered on each pull request & issue and emailling to all of us?

**Comment by SteveYurongSu** [comment]
@xwm1992 

This GitHub Action will only be triggered when: 

1. new users opening their first issues in this repo

2. new contributors submitting their first PRs in this repo

The GitHub Bot will automatically leave a comment which is defined in 'greetings.yml' for these issues / PRs. We will not be notified unless we subscribe to the mailing list that follows all issues / PRs.

**Pull request merged**